sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+- Azure DocumentTranslation client library for Java. This package contains Microsoft Azure DocumentTranslation client library.
```

Comment:
There seems to be a repetition. Please use [this blurb](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md#100-preview1-2019-06-28) for the initial beta release

Reply:
Updated